### DISCUSSION TITLE
Обновление списка сайтов

### DISCUSSION THREAD

**Discussion by shebaru** [xtrime-ru/antizapret-vpn-docker#9] [Q&A] [answered]
Добрый день. Несколько дней назад установил на vps антизапрет через ваш способ.
В докере и убунту в целом не силён совсем.
Подскажите, пожалуйста, обновляется ли список сайтов автоматически или как его правильно обновлять? 
Я так понимаю, что уже и ютуб пофиксили.
Спасибо

**Comment by xtrime-ru** [comment, marked as answer]
Привет! 

Есть 2 <strike>стула</strike> списка: 

1. Список заблокированных сайтов. Он обновляется автоматически каждые 6 часов
2. Список сайтов и ip адресов, добавленный вручную. Для случаев когда сайт не заблокирован, но должен работать через vpn. Например из за замедлений, или потому что сам сайт блокирует российские IP. Это youtube, chatgpt и тд. 
 
    Этот список нужно обновлять руками: добавляя домены в файл: config/include-hosts-custom.txt или делать git pull, что бы скачать новую версию. После обновления нужно перезапустить контейнер либо подождать до 6 часов.


Для того что бы сейчас получить последнюю версию нужно обновить репозиторий и пересобрать контейнер: 

```shell
cd %путь до папки с антизапретом%
git pull 
# если были изменения в файлах, то git pull не сработает. Нужно куда то сохранить изменения и сделать git fetch && git reset --hard origin/master
docker compose build
docker compose up -d
```

**Reply by shebaru** [reply]
Сработало. Спасибо.

**Reply by Murakumo-JP** [reply]
А чтоб исключить хост надо менять это в rootfs/root/antizapret/config/exclude-hosts-dist.txt ? или в config/exclude-hosts-custom.txt ? я проста хочу исключить spotify.com который был добавлен в последней обнове ибо теперь рекламу стали мне пихать.

**Reply by xtrime-ru** [reply]
@Murakumo-JP 
У вас видимо старый образ. В актуальной версии папки rootfs больше нет, что бы не вызывать путаницу. Добавлять  правила надо в папку ./config на хосте.

**Reply by Murakumo-JP** [reply]
Хм..... я вот только сегодня накотил новую версию и папка есть на сервере. но лодно буду знать спасибо.

**Reply by xtrime-ru** [reply]
А, вы имеете ввиду не в контейнере? Да, в репозитории папка есть, но она используется только для сборки образа. В работе она не используется. 

При работе используются только папки keys и config.

**Reply by Murakumo-JP** [reply]
Я переустанавливал убунту на сервере и ставил с нуля но папка почему то создалась.
![image](https://github.com/user-attachments/assets/84300cca-3de0-4c93-95f1-0521ef3bb926)

**Reply by Murakumo-JP** [reply]
> А, вы имеете ввиду не в контейнере? Да, в репозитории папка есть, но она используется только для сборки образа. В работе она не используется.
> 
> При работе используются только папки keys и config.

Спасибо понял вас.

**Reply by Murakumo-JP** [reply]
@xtrime-ru и ещё глупый вопрос а чтоб перезапустить контейнер надо использовать docker restart compose ?  проста боюсь что-то сломать случайно по не знанию

**Reply by xtrime-ru** [reply]
docker restart antizapret-vpn или docker compose restart 
Разницы нет. docker restart работает откуда угодно, а для работы docker compose нужно находится в папке где лежит docker-compose.yml файл.

**Reply by Murakumo-JP** [reply]
> docker restart antizapret-vpn или docker compose restart Разницы нет. docker restart работает откуда угодно, а для работы docker compose нужно находится в папке где лежит docker-compose.yml файл.

Большое вам спасибо
